docs/operate/reference/services/discovery/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Discovery Service"
+linkTitle: "Discovery"
+description: "The discovery service is used to discover resources on a machine."
+type: docs
+weight: 50
+no_list: true
+#icon: true
+#images: ["/services/icons/discovery.svg"]
+tags: ["discovery", "services"]
+no_service: true
+date: "2025-03-03"
+# updated: ""  # When the content was last entirely checked
+# SMEs: John N.
+modulescript: true
+---
+
+The discovery service is used to discover resources on a machine.
+If you are [creating a modular resource](/operate/get-started/other-hardware/) that depends on other resources that are discoverable in a systematic way, you can create a discovery service as part of your module to discover those resources.
+
+## Example usage
```

Comment:
This is all a bit abstract to me right now. What does it discover/return and how do I use it? Do I need to view that in the logs or does the UI just add a drop down? Since this is already covering example usage, can you walk through a specific vision service that does this? Then you can explain what the discovery service returns and how you can use it.

Reply:
Okay I added some more detail here. On Monday I'll check with John whether there's a better sort of example--right now there's rtsp and webcam, but not sure we want to go that specific?